vignettes/multi-outcomes-vignette.Rmd

```diff
@@ -0,0 +1,125 @@
+---
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Multi Outcomes AugSynth Vignette}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
```

Comment:
Could you build this vignette to get a markdown file and an html file?? Either `devtools::build_vigenettes()` or just using `knitr::knit()` should be fine.

Reply:
Added.